src/cmd/run.js

```diff
@@ -268,29 +273,33 @@ export class ExtensionRunner {
   profilePath: string;
   firefoxApp: typeof defaultFirefoxApp;
   firefox: string;
+  customPrefs: FirefoxPreferences;
 
   constructor(
     {
       firefoxApp, sourceDir, manifestData,
-      profilePath, firefox,
+      profilePath, firefox, customPrefs,
     }: ExtensionRunnerParams
   ) {
     this.sourceDir = sourceDir;
     this.manifestData = manifestData;
     this.profilePath = profilePath;
     this.firefoxApp = firefoxApp;
     this.firefox = firefox;
+    if (customPrefs) {
```

Comment:
Originally I've put the if statement there because flow doesn't like undefined assignment, now i've added default `customPrefs = {}` value instead.